.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -147,10 +98,59 @@ data class CIBuildModel (
+                    functionalTestsDependOnSpecificBuilds = true,
+                    omitsSlowProjects = true),
+            Stage("Branch Build Accept", "Run performance and functional tests (against distribution)",
+                    specificBuilds = listOf(
+                            SpecificBuild.BuildDistributions,
+                            SpecificBuild.Gradleception,
+                            SpecificBuild.SmokeTests),
+                    functionalTests = listOf(
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java7),
+                            TestCoverage(TestType.platform, OS.windows, JvmVersion.java8)),
+                    performanceTests = listOf(PerformanceTestType.test),
+                    omitsSlowProjects = true),
+            Stage("Master Accept", "Rerun tests in different environments / 3rd party components",
+                    trigger = Trigger.eachCommit,
+                    functionalTests = listOf(
+                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.linux, JvmVersion.java7),
+                            TestCoverage(TestType.quickFeedbackCrossVersion, OS.windows, JvmVersion.java7),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java10),
+                            TestCoverage(TestType.parallel, OS.linux, JvmVersion.java7, JvmVendor.ibm))),
+            Stage("Release Accept", "Once a day: Rerun tests in more environments",
+                    trigger = Trigger.daily,
+                    functionalTests = listOf(
+                            TestCoverage(TestType.soak, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.soak, OS.windows, JvmVersion.java8),
+                            TestCoverage(TestType.allVersionsCrossVersion, OS.linux, JvmVersion.java7),
+                            TestCoverage(TestType.allVersionsCrossVersion, OS.windows, JvmVersion.java7),
+                            TestCoverage(TestType.noDaemon, OS.linux, JvmVersion.java8),
+                            TestCoverage(TestType.noDaemon, OS.windows, JvmVersion.java8),
+                            TestCoverage(TestType.platform, OS.macos, JvmVersion.java8),
+                            TestCoverage(TestType.platform, OS.linux, JvmVersion.java9)),
+                    performanceTests = listOf(
+                            PerformanceTestType.experiment)),
+            Stage("Historical Performance", "Once a week: Run performance tests for multiple Gradle versions",
+                    trigger = Trigger.weekly,
+                    performanceTests = listOf(
+                            PerformanceTestType.historical)),
+            Stage("Experimental", "On demand: Run experimental tests",
+                    trigger = Trigger.never,
+                    runsIndependent = true,
+                    functionalTests = listOf(TestCoverage(TestType.platform, OS.linux, JvmVersion.java11))))
```

Comment:
Why was this moved down?

Reply:
Actually, it was only moved into the constructor, so that I could overwrite the values in the tests. I've restored the same ordering.